source/examples/generated/react-native/js/dictionary-test.snippet.query-objects-with-dictionary.jsx

```diff
@@ -4,17 +4,22 @@ const HomeList = () => {
 
   // run the `.filtered()` method on all the returned homeOwners to
   // find all homeOwners that have a house with a listed price
-  const listedPriceHomes = homeOwners.filtered('home.@keys = "price"');
+  const listedPriceHomes = useQuer(HomeOwner, homeOwners => {
+    return homeOwners.filtered('home.@keys = "price"');
+  });
 
   // run the `.filtered()` method on all the returned homeOwners to
   // find the house with the address "Summerhill St."
-  const summerHillHouse = homeOwners.filtered(
-    'home["address"] = "Summerhill St."',
-  )[0].home;
+  const summerHillHouse = useQuery(HomeOwner, homeOwners => {
+    return homeOwner.filtered('home["address"] = "Summerhill St."');
```

Comment:
```suggestion
    return homeOwners.filtered('home["address"] = "Summerhill St."');
```
Is this supposed to be ``homeOwners``?

Reply:
Yep! Thanks for catching that.